### DISCUSSION TITLE
routing.BaseRoute and BaseRoute

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [fastapi/fastapi#9716] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
na
```

### Description

What is the difference between routing.BaseRoute and BaseRoute in the routing.py module?

https://github.com/tiangolo/fastapi/blob/223ed676821f729cc31018c377279258087acecc/fastapi/routing.py#L495

https://github.com/tiangolo/fastapi/blob/223ed676821f729cc31018c377279258087acecc/fastapi/routing.py#L496

I'm getting the mypy error: "BaseRoute" has no attribute "path"  [attr-defined]

### Operating System

Windows

### FastAPI Version

0.95

### Python Version

3.10.8

**Comment by hmvp** [comment]
There is no difference, they resolve to the same class...

Afaik the issue is that BaseRoute (unlike Route from the same module) does not define `path`
